src/main/scala/firrtl/FileUtils.scala

```diff
@@ -149,6 +149,7 @@ object FileUtils {
   def getLinesResource(resourceName: String): Seq[String] = {
     val inputStream = getClass.getResourceAsStream(resourceName)
     val text = io.Source.fromInputStream(inputStream).getLines().toSeq
+    text.length  // This forces lazy buffer to reify, please suggest a better solution
```

Comment:
While I can't speak to correctness, I think using `toList` instead of `toSeq` should have a similar effect.